scripts/getting-started.sh

```diff
@@ -0,0 +1,127 @@
+    else 
+        echo "\n🍺 We will need git to be installed, installing."
+        brew install git
+    fi
+
+    if prompt "\n🍺 Install cmake, openssl and protobuf?"; then
+        brew install cmake openssl protobuf
+    else
+        echo "🍺 Assuming cmake, openssl and protobuf are present."
+    fi
+elif [ "$os_name" = "Linux" ]; then
+    # find the distro name in the release files
+    distro=$( cat /etc/*-release | tr '[:upper:]' '[:lower:]' | grep -Poi '(debian|ubuntu|arch|fedora|opensuse)' | uniq | head -n 1 )
+
+    if [ "$distro" = "ubuntu" ]; then
+        echo "\n🐧 Detected Ubuntu. Using apt to install dependencies."
+        sudo apt install --assume-yes git clang curl libssl-dev protobuf-compiler
+    elif [ "$distro" = "debian" ]; then
+        echo "\n🐧 Detected Debian. Using apt to install dependencies."
+        sudo apt install --assume-yes git clang curl libssl-dev llvm libudev-dev make protobuf-compiler
+    elif [ "$distro" = "arch" ]; then
+        echo "\n🐧 Detected Arch Linux. Using pacman to install dependencies."
+        pacman -Syu --needed --noconfirm curl git clang make protobuf
+    elif [ "$distro" = "fedora" ]; then
+        echo "\n🐧 Detected Fedora. Using dnf to install dependencies."
+        sudo dnf update
+        sudo dnf install clang curl git openssl-devel make protobuf-compiler
+    elif [ "$distro" = "opensuse" ]; then
+        echo "\n🐧 Detected openSUSE. Using zypper to install dependencies."
+        sudo zypper install clang curl git openssl-devel llvm-devel libudev-devel make protobuf
+    else
+        if prompt "\n🐧 Unknown Linux distribution. Unable to install dependencies. Continue anyway?"; then
+            echo "\n🐧 Proceeding with unknown linux distribution..."
+        else
+            exit 1
+        fi
+    fi
+else
+    echo "❌ Unknown operating system. Aborting."
+    exit 1
+fi
+
+# Check if rust is installed
+if command -v rustc >/dev/null 2>&1; then
+    echo "\n✅︎🦀 Rust already installed."
+else
+    if prompt "\n🦀 Rust is not installed. Install it?"; then
+        echo "🦀 Installing via rustup."
+        curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+    else
+        echo "Aborting."
+        exit 1
+    fi
+fi
+
+# Ensure that we have wasm support
+if prompt "\n🦀 Setup the Rust environment (e.g. WASM support)?"; then
+    echo "🦀 Setting up Rust environment."
+    rustup default stable
+    rustup update
+    rustup target add wasm32-unknown-unknown
+    rustup component add rust-src
+
+    rustup update nightly
+    rustup target add wasm32-unknown-unknown --toolchain nightly
+fi
+
+if [ -d "minimal-template" ]; then
+    echo "\n✅︎ minimal-template directory already exists. -> Entering."
+else
+    echo "\n↓ Let's grab the minimal template from github."
+    git clone https://github.com/paritytech/polkadot-sdk-minimal-template.git minimal-template
+fi
+cd minimal-template
+
+echo "\n⚙️ And finally, let's compile and get the node up and running."
```

Comment:
IMO we should separate the build and run. I expect people to do something different for a few minutes while the initial compile runs. 

After that it would be nice if they could come back to a prompt whether to start the node instead of a ton of logs.